package.json

```diff
@@ -1,26 +1,30 @@
 {
-  "name": "@porch/eslint-config-porch",
-  "version": "9.0.0",
-  "description": "Porch's eslint configuration.",
-  "main": "index.js",
-  "scripts": {
-    "lint": "eslint ."
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/porchdotcom/eslint-config-porch.git"
-  },
-  "author": "Porch <npm-maintainers@porch.com>",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/porchdotcom/eslint-config-porch/issues"
-  },
-  "homepage": "https://github.com/porchdotcom/eslint-config-porch#readme",
-  "dependencies": {
-    "babel-eslint": "^8.0.2",
-    "eslint": "^4.11.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-mocha": "^4.6.0",
-    "eslint-plugin-react": "^7.5.1"
-  }
+    "name": "@porch/eslint-config-porch",
+    "version": "9.0.0",
```

Comment:
You'll need to bump the version, as this package doesn't auto publish. Should probably be `9.1.0`

Reply:
using npm version ?